learnyounode/01-baby-steps.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+let sum = 0;
+for (let i = 2; i < process.argv.length; i++) {
+  sum += Number(process.argv[i]);
+};
+
+console.log(sum);
```

Comment:
This will work. If you're a fan of method chaining, it's possible to also solve this one in one line:
```
process.argv.slice(2).map((arg) => Number(arg)).reduce((a,b) => a+b)
```